data/data/manifests/bootkube/etcd-service.yaml

```diff
@@ -13,3 +13,6 @@ spec:
   - name: etcd
     port: 2379
     protocol: TCP
+  - name: etcd-metrics
+    port: 23790
+    protocol: TCP
```

Comment:
Should we drop this as a service for metrics? it really isn't useful they need to connect to each member explicitly. We will provide this via endpoint nodeName such as `https://ip-10-0-150-40.ec2.internal:23790`  @brancz ?

Reply:
keeping